samples/watchos/project.yml

```diff
@@ -0,0 +1,53 @@
+name: watchosSample
+options:
+  bundleIdPrefix: com.jetbrains.watchapp2
+settings:
+  DEVELOPMENT_TEAM: N462MKSJ7M
+  CODE_SIGN_IDENTITY: "iPhone Developer"
+  CODE_SIGN_STYLE: Automatic
+  MARKETING_VERSION: "1.0"
+  CURRENT_PROJECT_VERSION: "4"
+  SDKROOT: iphoneos
```

Comment:
Is this correct for watchOS project?

Reply:
Yes, moreover I spent whole day to find that out.